src/js/events.js

```diff
@@ -268,163 +255,159 @@ jQuery( document ).ready( function() {
-    });
-
+    } );
   } );
 
   // calculate and list rational approximations within user parameters
   jQuery( "#input_interval_to_approximate" ).change( function() {
     var interval = line_to_decimal( jQuery ( "#input_interval_to_approximate" ).val() );
-    
-        current_approximations.convergent_indicies = [];
-        current_approximations.numerators = [];
-        current_approximations.denominators = [];
-        current_approximations.ratios = [];
-        current_approximations.numerator_limits = [];
-        current_approximations.denominator_limits = [];
-        current_approximations.ratio_limits = [];
-
-        get_rational_approximations(interval, current_approximations.numerators, current_approximations.denominators, 999999,
-                                    current_approximations.convergent_indicies,
-                                    current_approximations.ratios,
-                                    current_approximations.numerator_limits,
-                                    current_approximations.denominator_limits,
-                                    current_approximations.ratio_limits);
-        modify_update_approximations();
-  });
-  
-  // recalculate approximations when scale degree changes
-  jQuery( "#input_scale_degree").change( function() {
-  jQuery( "#txt_tuning_data" ).val( jQuery( "#txt_tuning_data" ).val().trim() );
+    current_approximations.convergent_indicies = [];
+    current_approximations.numerators = [];
+    current_approximations.denominators = [];
+    current_approximations.ratios = [];
+    current_approximations.numerator_limits = [];
+    current_approximations.denominator_limits = [];
+    current_approximations.ratio_limits = [];
+
+    get_rational_approximations(
+	  interval, 
+	  current_approximations.numerators, 
+	  current_approximations.denominators, 
+	  999999,
+      current_approximations.convergent_indicies,
+      current_approximations.ratios,
+      current_approximations.numerator_limits,
+      current_approximations.denominator_limits,
+      current_approximations.ratio_limits
+	);
 
-  if ( isEmpty(jQuery( "#txt_tuning_data" ).val()) ) {
-    alert( "No tuning data to modify." );
-    return false;
-   }
+    modify_update_approximations();
+  } );
 
+  // recalculate approximations when scale degree changes
+  jQuery( "#input_scale_degree").change( function() {
+    jQuery( "#txt_tuning_data" ).val( jQuery( "#txt_tuning_data" ).val().trim() );
     var index = parseInt( jQuery( '#input_scale_degree' ).val() ) - 1;
-  var lines = document.getElementById("txt_tuning_data").value.split(newlineTest);
-  jQuery ( "#input_interval_to_approximate" ).val(lines[index]);
-  jQuery ( "#input_interval_to_approximate" ).trigger("change");
-
-  });
+    var lines = document.getElementById("txt_tuning_data").value.split(newlineTest);
+    jQuery ( "#input_interval_to_approximate" ).val(lines[index]);
+    jQuery ( "#input_interval_to_approximate" ).trigger("change");
```

Comment:
for future refactors: jQuery methods are chainable, so you can do `.val(lines[index]).trigger('change')` in one line.